### DISCUSSION TITLE
레벨3 챕터 운영  논의

### DISCUSSION THREAD

**Discussion by pocojang** [woowacourse/prolog#912] [Ideas]
> (챕터는 스쿼드와 별개로 운영되는 모임)

3기는 스프린트 별 선택 참여 / 4기는 자동 필수 참여

## :one: 목적

- 공통으로 적용될 기술과 프로세스를 논의하고 개선
- 온보딩 & 인수인계 리소스 최소화
- 기술 부채 해결

## :two: 운영 방법

- 독립적인 모임 일정으로 매주 월요일 20시 (3기~4기 모두 참여)
  - 활동 기간은 스프린트로 나누어 진행할 예정
- 챕터 모임에서 결정된 내용은 스쿼드에서 작업 진행시 반영될 수 있다.
  - 한 사람 혹은 특정 사람이 모든 리팩터링과 문서화를 진행하지 않는다.
  - 샘플을 하나 만들어 다른 사람들이 규칙을 따라서 진행할 수 있게 돕는다.

### 프론트엔드

- 문서화
- 디자인 리소스 정리
- 정적 타이핑
- 오류 리포트

### 백엔드

- 소나큐브
- 에러 채널 (모니터링)
- 인프라 적합성 검토

**Comment by sihyung92** [comment]
3기-웨지 / 참여 (하고싶은건 천천히 생각)

**Comment by Tanney-102** [comment]
3기 - 브콜 / 참여
코어 개선 쪽에 기여해보고 싶네요.
간단하게는 전반적으로 ts(혹은 다른 라이브러리라도)로 마이그레이션 한다거나 api 레이어를 잘 추상화해서 분리해본다던가

**Reply by Tanney-102** [reply]
브랜치 전략도 좀 고민해봐야할듯?

**Reply by euijinkk** [reply]
API 레이어 추상화 재밌을 것 같아요!

**Comment by bperhaps** [comment]
3기-손너잘 / 참여
배치 프로세스 적용, 데이터 중심의 플랫폼 설계에 기여해보고 싶습니다.
그 첫번째가 배치.

**Reply by her0807** [reply]
인기있는 게시글 자동 갱신 처리가 이부분 인가요 ?! 멋지네요 👍🏻

**Reply by bperhaps** [reply]
네 브라운의 계획으로는 아마 그쪽이 되지 않을까 생각되네요 :)

**Comment by 2unhyun** [comment]
3기 - 티케 / 참여
월요일 매번 참석이 어려울수는 있지만, 전반적인 파일구조를 새로 잡고 문서화하는데 기여하고 싶네요.
레거시 청산이 주 목적이 될것같습니다.
